PWGHF/TableProducer/candidateSelectorDsToKKPi.cxx

```diff
@@ -22,13 +22,20 @@
 
 #include "PWGHF/Core/HfHelper.h"
 #include "PWGHF/Core/HfMlResponse.h"
+#include "PWGHF/Core/HfMlResponseDsToKKPi.h"
 #include "PWGHF/DataModel/CandidateReconstructionTables.h"
 #include "PWGHF/DataModel/CandidateSelectionTables.h"
 
 using namespace o2;
 using namespace o2::analysis;
 using namespace o2::framework;
 
+/// Struct to extend TracksPid tables
+struct HfCandidateSelectorDplusToPiKPiExpressions {
```

Comment:
Thanks for underling this error, my inattention. I have just compiled the code locally because at the moment there isn't a model for the BDT ready and so I cannot test the workflow yet. However there are people that are already working on the preparation of the model and we have aimed to prepare the code in order to test it as soon as the model will be ready.

Reply:
Yes, you have to use those variables and also the order must be the same

Reply:
No, the purpose of HfMlResponseDsToKKPi is to be able to define the list of variables to use with a configurable. It contains all the possible variables we could want to use, then only a subset of them is effectively employed. In this [presentation](https://indico.cern.ch/event/1328768/contributions/5639840/attachments/2743423/4772971/d2h_meeting_ml_24_10_23.pdf) you can find more details.

Now I noticed this part is not implemented yet in your code. You can have a look at the Dplus selector, in particular [here](https://github.com/AliceO2Group/O2Physics/blob/87baf8614d59f8490d05bf6691f27b4ddb80b00f/PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx#L69) and [here](https://github.com/AliceO2Group/O2Physics/blob/87baf8614d59f8490d05bf6691f27b4ddb80b00f/PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx#L114-L125), to know how to proceed.

Reply:
Ok, got it, many thanks! So I'll add also this part